mantle/cosa/build.go

```diff
@@ -30,39 +30,42 @@ var (
 	ErrMetaFailsValidation = errors.New("meta.json failed schema validation")
 )
 
-func buildParser(r io.Reader) (*Build, error) {
+func buildParser(r io.Reader, validateAgainstSchema bool) (*Build, error) {
 	dec := json.NewDecoder(r)
 	dec.DisallowUnknownFields()
 	var cosaBuild *Build
 	if err := dec.Decode(&cosaBuild); err != nil {
 		return nil, errors.Wrapf(err, "failed to parse build")
 	}
+	if !validateAgainstSchema {
```

Comment:
nit: Seems like this should be inverted so we don't have two:
```go
return cosaBuild, nil
```
However, not worth blocking over :slightly_smiling_face: